src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,12 +2,17 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Method login was called");
         User user = findByLogin(login);
+        logger.info("Attempt to Log In using login: {}", login);
```

Comment:
the first message could be dropped in that case, because second one does essentially the same thing, also this looks more like a `trace` or `debug` level to me